xml/System.Net/IPAddress.xml

```diff
@@ -196,9 +196,22 @@
         <Parameter Name="address" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="address">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="address">The byte span value of the IP address.</param>
```

Comment:
What language do we use elsewhere when we take a `Span<byte>`? "byte span" sounds awkward to me. I might have used something like:

```suggestion
        <param name="address">The byte representation of the IP address, in network byte order.</param>
```

Reply:
@scalablecory David and I have been corrected by @rpetrusha many times to write "a byte span" instead of "a span of bytes", but maybe for some cases we could use the wording you suggested. What do you think, @rpetrusha?

Reply:
@mairaw - Please advise. There are probably some standard "Microsoft Tech Writing" rules that need to be observed here.

Reply:
I don't think we have any official rules re. this. Since type is shown automatically, we don't necessarily need to say what that represents in technical terms, so your suggestion looks good @scalablecory.

Otherwise, I'd go with @rpetrusha suggestion to say byte span when applicable.